app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    },
+    why: [{
+      who: ["provider"],
+      what: "uses this to verify a candidate’s identity. If offered a place, it is then used when making a DBS check"
+    }, {
+      who: ["provider", "dfe"],
+      what: "use a combination of this and other values to identify individual candidates and for record matching"
+    }, {
+      who: ["hesa"],
+      what: "requires this to facilitate “checking data with providers and for Statutory Customers to link student records collected by HESA for statistical purposes”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for proof of identification"],
+    notes: ["Ideally this field would be combined with family name but recorded separately to support legacy student record systems"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Family (last) name",
+  items: {
+    question: "Last name",
+    required: "Yes",
+    api: {
+      param: "candidate.last_name",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "uses this to verify a candidate’s identity. If offered a place, it is then used when making a DBS check"
+    }, {
+      who: ["provider", "dfe"],
+      what: "use a combination of this and other values to identify individual candidates and for record matching"
+    }, {
+      who: ["hesa"],
+      what: "requires this to facilitate “checking data with providers and for Statutory Customers to link student records collected by HESA for statistical purposes.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for proof of identification"],
+    notes: [
+      "Ideally this field would be combined with given name but recorded separately to support legacy student record systems.",
+      "HESA also ask for ‘Immediately prior surname’ to “to allow matching of records where a student has changed surname” but we don’t capture this information.",
+      "Should we allow changes post-submission?"
+    ]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Date of birth",
+  items: {
+    question: "Date of birth",
+    required: "Yes",
+    api: {
+      param: "candidate.date_of_birth",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "use this when performing a DBS check"
+    }, {
+      who: ["hesa"],
+      what: "needs this to be in the format YYYY-MM-DD in order to link to other records (e.g. higher education, wider than just ITT)"
+    }, {
+      who: ["dfe"],
+      what: "uses this for record matching and statistical analysis, e.g. looking at application rates of over 35 year olds (something the minister has historically been interested in)"
```

Comment:
GOV.UK style drops the dots from etc, eg, ie – but also recommends against them:
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#eg-etc-and-ie

Reply:
Updated and replaced all occurrences of e.g. and i.e.